pandas/core/groupby/groupby.py

```diff
@@ -581,6 +582,17 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if (name == 'hist' and
+                    kwargs.get('equal_bins', False) is True):
+                # GH-22222
+                bins = kwargs.get('bins')
+                if bins is None:
```

Comment:
What is the purpose of this? Doesn't the keyword already default to 10?

Reply:
The keyword defaults to 10 after the function is called (`curried_axis`), but we need its value before that (on line 593)